src/components/CreateCourse/CreateCourse.tsx

```diff
@@ -4,17 +4,29 @@ import Input from '../../common/Input/Input';
 import Button from '../../common/Button/Button';
 import getCourseDuration from '../../helpers/getCourseDuration';
 import { v1 as uuidv1 } from 'uuid';
+import { mockedAuthorsList } from '../../constants';
 
 const forbiddenSymbols = /[@#$%^&]/;
 
-const CreateCourse = ({ authors, createCourseButtonClicked, addNewAuthor }) => {
+const CreateCourse = () => {
 	const [title, setTitle] = useState('');
 	const [description, setDescription] = useState('');
-	const [addAuthorList, setAddAuthorList] = useState(authors);
+	const [addAuthorList, setAddAuthorList] = useState(mockedAuthorsList);
 	const [deleteAuthorList, setDeleteAuthorList] = useState([]);
 	const [author, setAuthor] = useState('');
 	const [duration, setDuration] = useState();
 
+	const createCourseButtonClicked = (newCourse) => {
+		// setCourses([...courses, newCourse]);
+		// setshowNewCourse((prev) => !prev);
+		console.log(newCourse);
```

Comment:
please remove all console logs that are not intended to be seen in 'production'

Reply:
I still see this console.log. 
What's the purpose of having it here? 
Or is it a work in progress?